CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to contbitute
+We are glad you are reading this, because we need volunteer works to improve vt-optimizer.
+
+## Testing
+Please write tests for every functionality.
+
+## Submitting changes
+Please send a [GitHub Pull Request to the dev branch](https://github.com/ibesora/vt-optimizer/pull/new/master), so that @ibesora can merge to master when everything is documented and is ready to go after passing all tests.
+
+## Coding conventions
+FIXME.
```

Comment:
I'm using eslint-config-geostart so maybe adding a link to that repo would be good https://github.com/geostarters/eslint-config-geostart. All the rules are written there and I don't think we should copy them here.

Reply:
Yes, I modified as advised.